diskutil.c

```diff
@@ -96,14 +113,21 @@ static void update_io_tick_disk(struct disk_util *du)
 	dus->s.ios[1] += (__dus.s.ios[1] - ldus->s.ios[1]);
 	dus->s.merges[0] += (__dus.s.merges[0] - ldus->s.merges[0]);
 	dus->s.merges[1] += (__dus.s.merges[1] - ldus->s.merges[1]);
-	dus->s.ticks[0] += (__dus.s.ticks[0] - ldus->s.ticks[0]);
-	dus->s.ticks[1] += (__dus.s.ticks[1] - ldus->s.ticks[1]);
-	dus->s.io_ticks += (__dus.s.io_ticks - ldus->s.io_ticks);
-	dus->s.time_in_queue += (__dus.s.time_in_queue - ldus->s.time_in_queue);
+	dus->s.ticks[0] += safe_32bit_diff(__dus.s.ticks[0], ldus->s.ticks[0]);
+	dus->s.ticks[1] += safe_32bit_diff(__dus.s.ticks[1], ldus->s.ticks[1]);
+	dus->s.io_ticks += safe_32bit_diff(__dus.s.io_ticks, ldus->s.io_ticks);
+	dus->s.time_in_queue +=
+			safe_32bit_diff(__dus.s.time_in_queue, ldus->s.time_in_queue);
 
 	fio_gettime(&t, NULL);
 	dus->s.msec += mtime_since(&du->time, &t);
 	memcpy(&du->time, &t, sizeof(t));
+
```

Comment:
Please remove this `#pragma` and change the memcpy() calls in this function into assignments. There is no reason to use memcpy() here.

Reply:
Done.